readme.md

```diff
@@ -74,6 +85,11 @@ The aim was to see, at its absolute minimum, how little code do you need to make
         <td>triggered after input and file validation - see 'File Error Types' section for more</td>
     </tr>
     <tr>
+        <td>lu:start</td>
```

Comment:
This event will always be called once per file selection, whereas the `lu:before` might get called before each file upload starts, depending on the `singleFileUploads` flag.